utils/update-checkout-config.json

```diff
@@ -290,22 +290,22 @@
         "tensorflow": {
             "aliases": ["tensorflow"],
             "repos": {
-                "llvm" : "04bdb56f3d90c394508b7259f86d5cca3c0edc15",
-                "clang" : "b44dbbdf44bf2b29967ab6a435dffcb280c8cfdc",
-                "swift" : "tensorflow",
-                "lldb" : "tensorflow",
-                "cmark": "d875488a6a95d5487b7c675f79a8dafef210a65f",
-                "llbuild": "1d48457963e550022393bab298688777b5ce3515",
-                "swiftpm": "232e413dff26525937e8ba13d3569be771de9c5b",
-                "compiler-rt": "db785ee6e527aae21aeec12ecfed6e06acaf8123",
-                "swift-corelibs-xctest": "01e6126d1cd1c5a81e7a1695487c09d57261577c",
-                "swift-corelibs-foundation" : "5ecb86c5ab6fe842e9fbbd2da052861fb57394c6",
-                "swift-corelibs-libdispatch" : "9295346d20e52c9ff4d82906fd85287d8636c542",
-                "swift-integration-tests": "01eecd5a83279635823e78101a538132784bc628",
-                "swift-xcode-playground-support": "4cca45ecd5050ef579bbb55076cd5d6ca4e60fbc",
-                "ninja": "253e94c1fa511704baeb61cf69995bbf09ba435e",
+                "compiler-rt": "231981e6604ea6c40e38af4f7e131b6c9046f91e", 
```

Comment:
Minor: would you mind reordering these to be in the original order?

Reply:
done